### ISSUE TITLE
How do you estimate number of users uploading keys

### ISSUE THREAD

**Issue by ndegendogo** [micb25/dka#23]
Wie berechnest / schätzt du eigentlich inzwischen die Anzahl der User, die DEK keys hochladen? Oder ist das eine Kennzahl, die das RKI inzwischen veröffentlicht?
Seit cws v 1.9 (?) / den neuen TRL Zuweisungen, wo 'days since symptoms' mit eingerechnet werden stimmt ja der 'naive' Ansatz mit dem TRL=6 nicht mehr?

**Comment by micb25** [comment]
Mit der Umstellung auf CWS v1.9 haben die Schätzungen für mich unplausible Werte ergeben (zu hoch). Zusätzlich gibt es mit der neuen Version des ENF auch ein Feld [`reportType`](https://developers.google.com/android/exposure-notifications/exposure-notifications-api#data-structures), was seltsamerweise für eine nicht unerhebliche Anzahl an Schlüsseln auf `RECURSIVE` gesetzt war, obwohl dies laut Dokumentation `reserved for future` sein soll.

Unter diesen Umständen habe ich mich dazu entschlossen, die Personenzahl eher konservativ (Minimum) zu schätzen, indem momentan die Anzahl der hochgeladenen TEKs durch 13 (maximale Anzahl hochladbarer TEKs) dividiert wird. Die reale Quote lag wohl eher bei ca. [11 TEKs / Person](https://ctt.pfstr.de/).

Was mich stört ist, dass es seit Monaten detaillierte tagesaktuelle [Backend-Daten](https://github.com/corona-warn-app/cwa-server/blob/1b78a88a3c4beb9039ca6619a1cb3c1da530373b/services/distribution/src/test/statistic_data.json) gibt, aber nicht offiziell herausgegeben werden. Stattdessen müssen die Schlüsseldaten täglich heruntergeladen und prozessiert werden mit zunehmend sinkender Genauigkeit. Ich hoffe, dass sich dies in [naher Zukunft](https://github.com/corona-warn-app/cwa-app-android/pull/2043) ändert, weil ich es als unproduktiv sehe, Daten abzuschätzen, die eigentlich an anderer Stelle präzise vorliegen und diese man nur veröffentlichen müsste.

Hier einmal ein Auszug aus diesen tagesaktuellen JSON-Daten:
```
{
   "date":"2020-10-29",
   "app_downloads_cum":21254005,
   "key_uploads":2558,
   "test_total":83583,
   "total_redeemed":1313,
   "total_not_redeemed":679,
   "labs_total":165,
   "labs_done":152,
   "infections":16774,
   "day_of_week":"Thu",
   "week_of_year":44
}
```

**Comment by ndegendogo** [comment]
Die Felder für den Report-Type sowie für den "days since onset of symptoms" hat man bei cwa "gekapert" und mit anderer Bedeutung hinterlegt (nämlich für die 8-stufige Codierung des TRL, die das RKI wohl auch mit der neuen ENF/ENS Version beibehalten wollte).
Thomas Klingbeil hat hierzu einen Vortrag im   CCC gehalten, wo er das erklärt.

**Comment by ndegendogo** [comment]
> Was mich stört ist, dass es seit Monaten detaillierte tagesaktuelle Backend-Daten gibt, aber nicht offiziell herausgegeben werden. 

Ja, da gebe ich dir 100% recht. Es ist ärgerlich; es wäre absolut kein großer Mehraufwand, und würde Transparenz und Vertrauen schaffen.

**Comment by ndegendogo** [comment]
Report type = recursive steht für TRL= 1 oder 2, je nachdem ob DSOS auf 1 oder 2 steht.
Ich habe das in den vergangenen 3 Wochen überprüft. Es ist tatsächlich konsistent und redundant codiert.

**Comment by ndegendogo** [comment]
See the newest release cwa 1.11 with statistics
